### PR TITLE
Migrate UpgradeBannerNotification.tsx to chakra

### PR DESCRIPTION
**Pull request by Mahmadabid**

### CODE REVIEW

**Review by pettinarip — CHANGES_REQUESTED** [review]
Thanks for the PR @Mahmadabid 💪🏼

File: src/components/UpgradeBannerNotification.tsx [hunk 1]
```diff
@@ -1,33 +1,22 @@
 import React from "react"
 import BannerNotification from "./BannerNotification"
-import Emoji from "./OldEmoji"
+import Emoji from "src/components/Emoji"
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Fixed import

File: src/components/UpgradeBannerNotification.tsx [hunk 2]
```diff
@@ -1,33 +1,22 @@
...
 import Link from "./Link"
-import styled from "@emotion/styled"
+import { chakra, Center } from "@chakra-ui/react"
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Import order is changed

File: src/components/UpgradeBannerNotification.tsx [hunk 3]
```diff
@@ -1,33 +1,22 @@
...
 
-const StyledBannerNotification = styled(BannerNotification)`
-  display: flex;
-  justify-content: center;
-`
+const StyledBannerNotification = chakra(BannerNotification)
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Removed Chakra wrap from Chakra components

File: src/components/UpgradeBannerNotification.tsx [hunk 4]
```diff
@@ -1,33 +1,22 @@
...
-const StyledEmoji = styled(Emoji)`
-  align-self: center;
-  margin-right: 1rem;
-  flex-shrink: 0;
-`
-
-const StyledBannerContent = styled.div`
-  align-self: center;
-`
+const StyledEmoji = chakra(Emoji)
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Removed Chakra wrap from Chakra components

File: src/components/UpgradeBannerNotification.tsx [hunk 5]
```diff
@@ -1,33 +1,22 @@
...
 
 const UpgradeBannerNotification: React.FC = () => (
-  <StyledBannerNotification shouldShow>
-    <StyledEmoji text=":megaphone:" />
-    <StyledBannerContent>
+  <StyledBannerNotification display="flex" justifyContent="center" shouldShow>
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Removed flex

File: src/components/UpgradeBannerNotification.tsx [hunk 6]
```diff
@@ -1,33 +1,22 @@
...
+    <StyledEmoji text=":megaphone:"  alignSelf="center" mr="1rem" flexShrink="0"/>
```
[the remark this answers is not in the document]
Comment (Mahmadabid) [reply src/components/UpgradeBannerNotification.tsx]: Changed according to Chakra scale system

**Review by pettinarip — APPROVED** [review]
Thanks for the PR @Mahmadabid I've pushed a few tiny styles fixes. The rest looks great.

**Comment by pettinarip** [comment]
@Mahmadabid be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @Mahmadabid for code

**Pull request merged**